src/components/ComponentSettingsModal.jsx

```diff
@@ -416,14 +416,23 @@ const ComponentSettingsModal = ({
                       className="w-100"
                       onChange={categoryDropdownHandler}
                       isDisabled={isCategoryDisabled}
-                      defaultValue={{
+                      placeholder={"Select a category or create a new category..."}
+                      defaultValue={originalCategory ? 
+                        {
                           value: originalCategory,
                           label: generateInitialCategoryLabel(originalCategory, isCategoryDisabled),
-                        }}
+                        }
+                        : null
+                      }
                       options={allCategories}
                     />
                   </div>
-                  <span className={elementStyles.error}>{errors.category}</span>
+                  { errors.category &&
```

Comment:
We need to do this for the permalink field as well 
<img width="541" alt="Screenshot 2020-12-21 at 2 44 27 PM" src="https://user-images.githubusercontent.com/31984694/102747422-3a2dbd00-439b-11eb-97c4-a9fff5881835.png">

Reply:
Good catch, resolved in https://github.com/isomerpages/isomercms-frontend/pull/299/commits/b215b7e3dbfcf6738a61693d140e42d941832afc